shared/gh/api/gh.api.util.js

```diff
@@ -16,6 +16,17 @@
 define(['exports', 'moment', 'sinon', 'bootstrap-notify'], function(exports, moment, sinon) {
 
 
+    /////////////////
+    //  CONSTANTS  //
+    /////////////////
+
+    var periods = {
```

Comment:
Can you add a comment along the lines of `Keep track of number of milliseconds in a day, week and month`